tests/common/snappi_tests/snappi_fixtures.py

```diff
@@ -1524,13 +1525,21 @@ def gen_data_flow_dest_ip(addr, dut=None, intf=None, namespace=None, setup=True)
         arp_opt = f"-s {addr} aa:bb:cc:dd:ee:ff"
     else:
         arp_opt = f"-d {addr}"
+        is_tear_down = True
 
     try:
-        dut.shell(f"sudo {asic_arg} arp {int_arg} {arp_opt}")
-        dut.shell(
-            "{} config route {} prefix {}/32 nexthop {} {}".format(
-                asic_arg, cmd, DEST_TO_GATEWAY_MAP[addr]['dest'], addr,
-                DEST_TO_GATEWAY_MAP[addr]['intf']))
+        if not is_tear_down:
```

Comment:
Thanks for handling this. This could be simply "if setup:"

Reply:
Thanks @rraghav-cisco , adjusted